docs/src/pages/manual.md

```diff
@@ -60,8 +60,8 @@ using ResultTypes
 data = [1, 2, 3]
 
 ctx = @dispatch_context begin
-    a = @op 1 + 2
-	x = @op a + 3
+	a = @op 1 + 2
+ 	x = @op a + 3
```

Comment:
This seems to have been all mixed up. Should use spaces and not tabs, and this block should have consistent indentation.